pufferlib/ocean/trash_pickup/trash_pickup.py

```diff
@@ -0,0 +1,79 @@
+import numpy as np
+from gymnasium import spaces
+
+import pufferlib
+from pufferlib.ocean.trash_pickup.cy_trash_pickup import CyTrashPickup
+
+
+class TrashPickupEnv(pufferlib.PufferEnv):
+    def __init__(self, num_envs=1, render_mode=None, report_interval=1, buf=None, grid_size=10, num_agents=3, num_trash=15, num_bins=2, max_steps=300):
+        # Env Setup
+        self.render_mode = render_mode
+        self.report_interval = report_interval
+
+        # Validate num_agents
+        if not isinstance(num_agents, int) or num_agents <= 0:
```

Comment:
Cython should do the type checks here for you. Fine to validate negative if you really want to, but if you pass a negative number here, you sort of deserve it

Reply:
I think I'll leave this for myself because I am the type of person to make the "you sort of deserve it" mistake lol.